slasher/db/kv/backup_test.go

```diff
@@ -0,0 +1,20 @@
+package kv
+
+import (
+	"context"
+	"io/ioutil"
+	"path"
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/shared/testutil/require"
+)
+
+func TestStore_Backup(t *testing.T) {
```

Comment:
It would be nice to read something from the backup to check if it's not corrupted.